omnibus/package-scripts/agent/preinst

```diff
@@ -45,14 +45,8 @@ if [ "$DISTRIBUTION" != "Darwin" ]; then
         fi
     fi
 
-    PIP_PATH=$INSTALL_DIR/embedded/bin/pip
-    if [ -x $PIP_PATH ]; then
-        echo "Resetting integrations..."
-        $PIP_PATH freeze | grep ^datadog- | grep -v datadog-checks-base | xargs $PIP_PATH uninstall -y -q --no-cache-dir || true
-    fi
-
     # For versions < 6.10 using the custom datadog-pip, TUF and in-toto files were kept in TUF_REPO_DIR.
-    # We need to clean that up on upgrade now, since it was only removed on uninstall.
+    # They were never being cleaned up, so let's do that now on install/upgrade
```

Comment:
`They were not being cleaned up by these versions, so [...]` sounds a bit clearer to me